sandy/core/cov.py

```diff
@@ -891,7 +936,7 @@ def _gls_general_sensitivity(self, S, Vy_extra=None, sparse=False,
         index, columns = pd.DataFrame(S).columns, pd.DataFrame(S).index
         S_ = pd.DataFrame(S).values
         # GLS_sensitivity:
-        G_inv = self._gls_G_inv(S, Vy_extra=Vy_extra, sparse=sparse).values
+        G_inv = self._gls_G_inv(S, Vy_extra, sparse=sparse).values

```

Comment:
I preferred the previous line `G_inv = self._gls_G_inv(S, Vy_extra=Vy_extra, sparse=sparse).values`